wrangler-storage/src/main/java/io/cdap/wrangler/dataset/schema/SchemaFile.java

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.wrangler.dataset.schema;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import org.apache.commons.lang.builder.EqualsBuilder;
+
+import java.util.Map;
+
+/**
+ * The deserialized representation of a JSONSchema file.
+ */
+public class SchemaFile {
+  private String description;
+  private Discriminator discriminator;
+  private MetaData metadata;
+  private Map<String, JsonElement> definitions;
```

Comment:
JsonElement -> Object. We want to avoid gson classes in any of the proto classes, as it can easily cause clashes in dependent projects.